google/cloud/spanner_v1/session.py

```diff
@@ -63,12 +63,13 @@ class Session(object):
     _session_id = None
     _transaction = None
 
-    def __init__(self, database, labels=None, database_role=None):
+    def __init__(self, database, labels=None, database_role=None, observability_options=None):
```

Comment:
Just to clarify is this method used internally or by customers?
If only used internally, I don't see that we are passing observability_options to this method that is received through spanner client.